contrib/kafka/pkg/apis/sources/v1alpha1/kafka_types.go

```diff
@@ -66,9 +66,9 @@ type KafkaSourceNetSpec struct {
 
 // KafkaSourceSpec defines the desired state of the KafkaSource.
 type KafkaSourceSpec struct {
-	// Brokers are the Kafka servers the consumer will connect to.
+	// Bootstrap servers are the Kafka servers the consumer will connect to.
 	// +required
-	Brokers string `json:"brokers"`
+	BootstrapServers string `json:"bootstrapServers"`
```

Comment:
We haven't released this EventSource yet, correct?

Reply:
Nope, we haven't release it yet.